src/components/Search/Search.scss

```diff
@@ -32,6 +32,7 @@
     &.wkp-search-button-clear {
       background: none;
       color: #666;
+      right: 48px;
```

Comment:
@edellucien  the buttons clear and search are inside   flex container for the positionning .  I think you can remove the absoulte,right,top  positionning  instead of adding this css. But maybe I miss something